packages/wallets/src/utils/passwordUtils.ts

```diff
@@ -1,25 +1,8 @@
-import { zxcvbn, zxcvbnOptions } from '@zxcvbn-ts/core';
-import { dictionary } from '@zxcvbn-ts/language-common';
+/* eslint @typescript-eslint/triple-slash-reference: "off" */
+/// <reference path="../../src/@types/@contentpass/zxcvbn/contentpass-zxcvbn-config.d.ts" />
 import { passwordErrorMessage, passwordRegex, passwordValues, warningMessages } from '../constants/passwordConstants';
 
 export type Score = 0 | 1 | 2 | 3 | 4;
-export type passwordKeys =
```

Comment:
There's an error about `passwordKeys` are being imported in `passwordConstants.ts` file here bellow